jenkins-scripts/docker/reprepro_updater.bash

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -x
+
+set -e
+
+# Knowing Script dir beware of symlink
+[[ -L "${0}" ]] && SCRIPT_DIR=$(readlink "${0}") || SCRIPT_DIR="${0}"
+SCRIPT_DIR="${SCRIPT_DIR%/*}"
+
+PYTHON_VENV="${WORKSPACE}/venv"
+REPREPRO_GIT_REPO="https://github.com/j-rivero/reprepro-updater"
+REPREPRO_GIT_BRANCH="${REPREPRO_GIT_BRANCH:-refactor}"
+REPREPRO_REPO_PATH="$WORKSPACE/repo"
+
+REPREPRO_PARAMS=''
+if ${COMMIT}; then
+    REPREPRO_PARAMS='--commit'
+fi
+
+if [[ ! -f "${HOME}/.buildfarm/reprepro-updater.ini" ]]; then
+    echo "The reprepro configuration file was not found"
+    exit -1
```

Comment:
Does -1 / 255 have any specific significance as an exit code?

Reply:
no. I use it as a generic failure.